examples/solidity/ERC20/erc20.qnt

```diff
@@ -147,7 +147,7 @@ module erc20 {
     balanceOf' = balanceOf,
     val failed = or {
      tx.value < 0,
-     tx.sender > tx.spender,
+     tx.sender == tx.spender,
```

Comment:
This is a fix for a type error that was not flagged by the type checker before, but is now. The reason why this PR change impacted this is not quite clear to me yet. However, @konnov reported a similar bug this morning which is not solved by this, so I'll further investigate this behavior when looking at that.